src/components/Post/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.scss';
+
+const Post = ({
+  title,
+  body,
+  publishedAt,
+  author: { name },
+}) => (
+  <div className="post">
+    <div className="post__header">
+      <div className="post__header__user">
+        <div className="post__header__username">
+          <div className="nickname">{`@${name}`}</div>
+        </div>
+      </div>
+    </div>
```

Comment:
When you see something like this is when we should ask the question "Are we using divs we don't really need?" Usually the answer is yes. If it's the only thing nested you don't really need the nestings